pkg/local-gadget-manager/local-gadget-manager.go

```diff
@@ -91,17 +91,15 @@ func NewManager(runtimes []*containerutils.RuntimeConfig) (*LocalGadgetManager,
 	if err != nil {
 		return nil, fmt.Errorf("error creating containers map: %w", err)
 	}
-	containerEventFuncs := []containercollection.FuncNotify{}
-	containerEventFuncs = append(containerEventFuncs, l.containersMap.ContainersMapUpdater())
-	containerEventFuncs = append(containerEventFuncs, l.tracerCollection.TracerMapsUpdater())
 
 	err = l.ContainerCollection.Initialize(
-		containercollection.WithPubSub(containerEventFuncs...),
+		containercollection.WithPubSub(l.containersMap.ContainersMapUpdater()),
```

Comment:
I don't think the localgadgetmanager needs the ContainerMap.

Reply:
It's not needed even in kubectl-gadget because the audit seccomp gadget doesn't use this anymore after this PR. However I plan to remove that in a following PR to make reviewing this easier.

Reply:
But for the gadgettracermanager it is used by standard tools.

Reply:
Ah, I almost forgot we still have those.